mne/io/egi/egimff.py

```diff
@@ -544,6 +548,13 @@ def _read_segment_file(self, data, idx, fi, start, stop, cals, mult):
 
                 # Start reading samples
                 while samples_to_read > 0:
+                    if samples_to_read == 1 and fid.tell() == file_size:
+                        # We are in the presense of the EEG bug
+                        # fill with zeros and break the loop
+                        data_view = data[n_data1_channels:, -1] = 0
+                        logger.warning('This file has the EGI PSG sample bug')
```

Comment:
For FIF when acquisition is started and stopped, we both set it to zero *and* add an annotation `BAD_ACQ_SKIP`. We should probably do the same thing here. The `BAD_ACQ_SKIP` might be a bit of a misnomer, though, so maybe label it `BAD_EGI_PSG`?

Reply:
This should be `warn` from `mne.utils` instead of `logger.warn`